packages/clerk-js/src/ui/components/UserProfile/DeleteUserForm.tsx

```diff
@@ -36,7 +38,12 @@ export const DeleteUserForm = withCardStateProvider((props: DeleteUserFormProps)
       }
 
       await user.delete();
-      await navigateAfterSignOut();
+
+      // TODO: Investigate if we need to call `setActive` with {session: null}
+      if (otherSessions.length === 0) {
+        return navigateAfterSignOut();
+      }
+      await navigateAfterMultiSessionSingleSignOutUrl();
```

Comment:
💭  I wouldn't necessarily expect that this method would need to be aware of multi-session. I understand why it was done this way as part of this PR, but it feels like a better internal abstraction might be to consolidate the multi-session logic into `navigateAfterSignOut()`.

Reply:
I think creating an abstraction here would make our lifer harder, we have `useMultisessionActions` which needs to know the difference between single session sign out and multisession sign out. We can refactor but i don't see a clear benefit here.

Reply:
Benefit would be not having to spread some of this logic across many different methods. That's fine to leave it for now, let's jut keep an eye on these code paths and we can abstract if it feels like multi-session logic is too difficult to manage.